core/src/integration/java/com/radixdlt/integration/steady_state/simulation/consensus_ledger/TimeoutPreviousVoteWithDroppedProposalsTest.java

```diff
@@ -139,9 +139,9 @@ public void sanity_test() {
               final var timeoutsSent = (long) metrics.bft().pacemaker().timeoutsSent().get();
               final var timedOutRounds = (long) metrics.bft().pacemaker().timedOutRounds().get();
 
-              // this ensures that we only need a maximum of two local timeouts per round
+              // this ensures that we only need a maximum of three local timeouts per round
```

Comment:
Can we explain why we need three in the comment? I guess we get another timeout because of the extra new delay we introduced?

I worry this assertion might be a little flaky.
Also - the below comment on line 143 is now inaccurate.

Reply:
> I guess we get another timeout because of the extra new delay we introduced?
Yes.

I've added a comment. Hmh, I don't think it's _that_ flaky :D